packages/seacas/README.md

```diff
@@ -1,206 +1,16 @@
-[![Analysis Status](https://scan.coverity.com/projects/2205/badge.svg?flat=1)](https://scan.coverity.com/projects/gsjaardema-seacas)
-[![Spack Version](https://img.shields.io/spack/v/adios2.svg)](https://spack.readthedocs.io/en/latest/package_list.html#seacas)
-[![Appveyor Build](https://ci.appveyor.com/api/projects/status/pis4gok72yh0wwfs/branch/master?svg=true)](https://ci.appveyor.com/project/gsjaardema/seacas/branch/master)
-[![Github Actions -- CI Serial](https://github.com/sandialabs/seacas/actions/workflows/build_test.yml/badge.svg)](https://github.com/sandialabs/seacas)
-[![Github Actions -- CI Variants](https://github.com/sandialabs/seacas/actions/workflows/build_variant.yml/badge.svg)](https://github.com/sandialabs/seacas)
-[![Github Actions -- CI Intel](https://github.com/sandialabs/seacas/actions/workflows/intel-build.yml/badge.svg)](https://github.com/sandialabs/seacas)
-[![Github Actions -- CI MSYS2](https://github.com/sandialabs/seacas/actions/workflows/msys2.yml/badge.svg)](https://github.com/sandialabs/seacas)
+## Documentation
+See SEACAS (including Exodus) documentation at http://sandialabs.github.io/seacas-docs/
 
-*  [Get the sources](#get-the-sources)
-*  [Build instructions](#build-instructions)
-*  [Configure, Build, and Install SEACAS](#configure-build-and-install-seacas)
-*  [Parallel Build](#parallel-build)
-*  [Testing](#testing)
-*  [Exodus](#exodus)
-*  [Trilinos](#trilinos)
-*  [SPACK](#spack)
-*  [License](#license)
-*  [Contact information](#contact-information)
-*  NOTE: The old imake-based build has been removed.
+## Installation
+Installation directions are available at http://github.com/sandialabs/seacas
 
-## Get the sources
-```sh
-git clone https://github.com/sandialabs/seacas.git
-```
-This will create a directory that will be referred to as _seacas_ in
-the instructions that follow. You can rename this directory to any
-other name you desire. Set an environment variable pointing to this
-location by doing:
+## Contact
 
-```sh
-cd seacas && export ACCESS=`pwd`
-```
+Greg Sjaardema,
+Sandia National Laboratories,
+<gdsjaar@sandia.gov>, <gsjaardema@gmail.com>
 
-## Build instructions
+We appreciate feedback from users of this package.  Please send
+comments, suggestions, and bug reports to Greg Sjaardema
+<gdsjaar@sandia.gov>, <gsjaardema@gmail.com>
 
-### Automatically download and build dependencies (Third-Party Libraries)
-
-There are a few externally developed third-party libraries (TPL) that
-are required (or optional) to build SEACAS: HDF5, NetCDF, CGNS, MatIO,
-Kokkos, and (if MPI set) PnetCDF libraries. You can build the
-libraries using the `install-tpl.sh` script, or you can install them
-manually as detailed in
-[TPL-Manual-Install.md](TPL-Manual-Install.md).
```

Comment:
Looks like you removed the entire discussion of TPL dependencies and how to install them from the README file. I didn't see this moved to a new file but may have missed it. Is the TPL dependency information available somewhere else?